### DISCUSSION TITLE
GPT4 8k context

### DISCUSSION THREAD

**Discussion by arminta7** [brianpetro/obsidian-smart-connections#158] [Q&A] [answered]
Does the GPT4 option take advantage of the longer 8k context?

**Comment by brianpetro** [comment, marked as answer]
Hey @arminta7 

Not yet. 

One of my top priorities is better utilizing the available context window. So it should be coming very soon.

If you're only using GPT-4, you can change this line https://github.com/brianpetro/obsidian-smart-connections/blob/d5d79d3f0abb5976740c2c62852928af8a2d070b/main.js#L3412 from `10000` to `20000` to utilize the GPT-4 increase in context size.

I hope that helps,
Brian 🌴

**Reply by arminta7** [reply]
Sounds great. Thanks!
